src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets

```diff
@@ -326,7 +326,18 @@
     </ItemGroup>
 
     <WriteItemsToJson JsonFileName="%(HelixWorkItemList.BuildCompleteJson)" Items="@(BuildComplete)" />
-    <Message Text="Wrote build complete JSON for @(BuildComplete->Count()) Queues." />
+    <Message Text="Wrote job-start (build complete) JSON for @(BuildComplete->Count()) Queues." />
+	
+    <CreateItem Include="%(HelixWorkItemList.BuildCompleteJson)" AdditionalMetadata="RelativeBlobPath=JobStartJsonMessages.json">
+      <Output TaskParameter="Include" ItemName="JobStartJsons" />
+    </CreateItem>
```

Comment:
Here's what I'm thinking would do the same thing:

```xml
<ItemGroup>
  <JobStartJsons Include="%(HelixWorkItemList.BuildCompleteJson)">
    <RelativeBlobPath>JobStartJsonMessages.json</RelativeBlobPath>
  </JobStartJsons>
</ItemGroup>
```

Is it any different? This is what I'm used to.